game/src/graphics/dmg_numbers.rs

```diff
@@ -28,10 +28,8 @@ struct DmgNumber(Vec3, f64);
 fn instance(
     mut commands: Commands,
     asset_server: Res<AssetServer>,
-    entity_with_hp: Query<
-        (&GlobalTransform, Option<&Collider>),
-        Without<Player>,
-    >,
+    enemy_query: Query<(&GlobalTransform, Option<&Collider>), Without<Player>>,
```

Comment:
`Attack` entities have a Transform and Collider. So this query might be picking up more entities than desirable. I'd go for `With<Enemy>` here.